### PR TITLE
Prepare for 2.5.0 release

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: conf/GeoIP.conf.default [hunk 1]
```diff
@@ -2,15 +2,15 @@
 # on setting up geoipupdate, including information on how to download a
 # pre-filled GeoIP.conf file.
 
-# Enter your user ID and license key below. These are available from
+# Enter your account ID and license key below. These are available from
 # https://www.maxmind.com/en/my_license_key. If you are only using free
 # GeoLite databases, you make leave the 0 values.
```
Comment (horgh) [inline conf/GeoIP.conf.default]: Pre-existing, but: s/make/may

File: dev-bin/clang-format-all.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+format="clang-format -i bin/geoipupdate.c -style=file"
```
Comment (horgh) [inline dev-bin/clang-format-all.sh]: Do we need the `bin/geoipupdate.c` given here?

**Review by horgh — CHANGES_REQUESTED** [review]
Looks great. clang-format even reordered headers, awesome! I have two comments.

**Comment by oschwald** [comment]
Thanks. I fixed both of those issues.

**Pull request merged**